### PR TITLE
Allow to change the location of home zone in zone editor

### PR DESCRIPTION
**Pull request by bramkragten**
On mobile there is no way to change the home location now in the zones editor, should that link to core or should we change the dialog in the zone editor?

### CODE REVIEW

File: src/panels/config/zone/ha-config-zone.ts [hunk 1]
```diff
@@ -233,6 +238,9 @@ export class HaConfigZone extends SubscribeMixin(LitElement) {
 
   protected firstUpdated(changedProps: PropertyValues) {
     super.firstUpdated(changedProps);
+    this._canEditCore = ["storage", "default"].includes(
```
Comment (balloob) [inline src/panels/config/zone/ha-config-zone.ts]: And is admin

File: src/translations/en.json [hunk 2]
```diff
@@ -1406,7 +1406,7 @@
           "add_zone": "Add Zone",
           "confirm_delete": "Are you sure you want to delete this zone?",
           "configured_in_yaml": "Zones configured via configuration.yaml cannot be edited via the UI.",
-          "edit_home_zone": "The location of your home can be changed in the general config.",
+          "edit_home_zone": "The radius of the Home zone can't be edited from the frontend yet. You can drag the marker on the big map.",
```
Comment (balloob) [inline src/translations/en.json]:
```suggestion
          "edit_home_zone": "The radius of the Home zone can't be edited from the frontend yet. Drag the market on the map to move the home zone.",
```

**Review by balloob — APPROVED** [review]

File: src/panels/config/zone/ha-config-zone.ts (hunk 1 above)
Comment (bramkragten) [reply src/panels/config/zone/ha-config-zone.ts]: You can't access zone config if you're not an admin, right?

Comment (balloob) [reply src/panels/config/zone/ha-config-zone.ts]: Right !

**Pull request merged**